### PR TITLE
[docs] Add description for multitenancy manager

### PR DESCRIPTION
**Pull request by Lada7878**
## Description

Add description for multitenancy manager.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Add description for multitenancy manager.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

File: modules/160-multitenancy-manager/docs/README_RU.md [hunk 1]
```diff
@@ -45,3 +48,12 @@ description: Мультитенантность и проекты в Kubernetes.
 1. По очереди создаются все ресурсы, описанные в шаблоне.
 
 > **Внимание!** При изменении шаблона проекта, все созданные проекты будут обновлены в соответствии с новым шаблоном.
+
+### Изоляция проекта
+
+В основе проекта используется механизм `Namespace`. Это своего рода контейнер, в котором можно группировать поды, сервисы, секреты и прочее, но полноценной изоляции он не обеспечивает. Для ее осуществления вы можете использовать дополнительные инструменты, которые обеспечивают детальный уровень контроля и безопасности для проекта, изолируя его от других и гарантируя, что используется только необходимое количество ресурсов:
```
Comment (z9r5) [inline modules/160-multitenancy-manager/docs/README_RU.md:54]:
```suggestion
В основе проекта используется механизм изоляции ресурсов в рамках пространства имен (Namespace). Пространства имен позволяют группировать поды, сервисы, секреты и другие объекты, но не обеспечивают полноценной изоляции. Проект позволяет дополнить пространства имен дополнительными инструментами изоляции, которые могут дать более высокий уровень контроля и безопасности. Для управления уровнем изоляции проекта можно использовать любые ресурсы Kubernetes, например:
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10948](https://github.com/deckhouse/deckhouse/pull/10948) to the branch [release-1.66](https://github.com/deckhouse/deckhouse/tree/release-1.66) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10949](https://github.com/deckhouse/deckhouse/pull/10949) to the branch [release-1.65](https://github.com/deckhouse/deckhouse/tree/release-1.65) successful!

**Pull request merged**